ui/app/components/secure-variable-form.js

```diff
@@ -16,6 +21,20 @@ export default class SecureVariableFormComponent extends Component {
     return !this.args.model?.path;
   }
 
+  @action
+  validatePath(e) {
+    const value = trimPath([e.target.value]);
+    let existingVariable = this.store
+      .peekAll('variable')
+      .without(this.args.model)
+      .find((v) => v.path === value);
```

Comment:
Is `this.store` isolated by namespace? If not, I think this might collide unintentionally if we have the same path in two different namespaces.

Reply:
Agreed, `this.store` is not isolated by namespace. We create a special property on jobs to help us avoid this by taking the name of the job and its namespace and storing that as the `id` (e.g. `"['app', 'default']"`).

Reply:
`praise`:  great job avoiding a network request with `peekAll`.

Reply:
Yep, you're both absolutely right.

I'll be taking a "namespacifying" pass at all Secure Variables routes soon. Figured I'd get the base logic in first.